sdk/src/main/kotlin/com/processout/sdk/core/logger/POLogger.kt

```diff
@@ -1,49 +1,61 @@
 package com.processout.sdk.core.logger
 
-internal class POLogger private constructor() {
+import com.processout.sdk.core.annotation.ProcessOutInternalApi
+
+@ProcessOutInternalApi
+class POLogger private constructor() {
 
     companion object {
+        internal const val ATTRIBUTE_LINE = "Line"
+        internal const val ATTRIBUTE_INVOICE_ID = "InvoiceId"
+        internal const val ATTRIBUTE_GATEWAY_CONFIGURATION_ID = "GatewayConfigurationId"
```

Comment:
Do we really need those constants defined here? It causes no harm, but logger resides in core layer, while invoice and gateway entities are existing in layer above it.

Maybe it's possible to move them to extension that could be defined where used or something similar?